src/plugins/irc/irc-server.h

```diff
@@ -93,6 +93,7 @@ enum t_irc_server_option
     IRC_SERVER_OPTION_NOTIFY,               /* notify list                   */
     IRC_SERVER_OPTION_SPLIT_MSG_MAX_LENGTH, /* max length of messages        */
     IRC_SERVER_OPTION_CHARSET_MESSAGE,      /* what to decode/encode in msg  */
+    IRC_SERVER_OPTION_SSL_PASSWORD,   /* client ssl certificate key password */
```

Comment:
Do new IRC server options need to be added to the end of this enum?

I did it just in case but if it's not necessary then it'd make sense to move this option up where `ssl_cert` is for better grouping. In most other places I put `ssl_password` right after `ssl_cert`.

Reply:
No, you can put the option where you want, so better move it with other SSL options.
But in irc-server.c, you must put the default value at the good position.

Reply:
In irc-server.c there's the name of option anyway now, but better move it as well to be consistent.